src/RetryAnnotationTrait.php

```diff
@@ -23,46 +26,39 @@ private function getRetryAttemptsAnnotation()
             $retries = $annotations['class']['retryAttempts'][0];
         }
 
-        return $this->validateRetryAttemptsAnnotation($retries);
+        return $this->parseRetryAttemptsAnnotation($retries);
```

Comment:
rename some `validate*()` into `parse*()` since IMO it communicates better the intend: `validate` kinda implies it validates, not return the normalized value